src/FINDOLOGIC/Export/Data/Property.php

```diff
@@ -11,13 +11,33 @@ public function __construct($key, $usergroup)
     }
 }
 
+class PropertyKeyNotAllowedException extends \RuntimeException
+{
+    public function __construct($key)
+    {
+        parent::__construct(sprintf('Property key "%s" is not allowed.', $key));
```

Comment:
The message gives little insight on why a property key may not be used. It should be hinted that such a key is reserved for internal use and would be overwritten when importing.

Reply:
Done